gossip/src/low_pass_filter.rs

```diff
@@ -0,0 +1,62 @@
+//! Fixed-point IIR filter for smoothing `alpha` updates.
+//!
+//! Implements:
+//!   alpha_new = K * target + (1 - K) * previous
+//!
+//! All math is unsigned integer fixed-point with `SCALE = 1000`
+//!
+//! The filter constant K is derived from:
+//!     K = W_C / (1 + W_C), where Wc = 2π * Fs / Tc
+//!     Fc = 1 / TC  (cutoff frequency)
+//!     Fs = 1 / refresh interval
+use std::num::NonZeroU64;
+
+// Fixed point scale for K and `alpha` calculation
+pub const SCALE: NonZeroU64 = NonZeroU64::new(1000).unwrap();
+// 2 * pi * SCALE
+const TWO_PI_SCALED: u64 = 6_283;
```

Comment:
```suggestion
const TWO_PI_SCALED: u64 = (std::f64::consts::PI * 2.0 * 1000.0) as u64;
```

Reply:
this suggestion leaves rounding to the platform. prefer a literal

Reply:
This does not round on the platform. The rounding is done at compile time and is 100% reproducible, else it would not build.

Reply:
> the rounding is 100% deterministic.
thanks for completely ignoring "to the platform"